Parsers/ASimDns/Parsers/ASimDnsMicrosoftNXlog.yaml

```diff
@@ -170,20 +227,23 @@ ParserQuery: |
       DnsQueryType=QTYPE_s,
       DnsResponseCode=RCODE_s,
       DnsResponseName=PacketData_s,
-      Dvc=Hostname_s,
```

Comment:
hi @jszigetvari-nxlog I just noticed that we have also changed two existing field mappings? Any specific reason for this change?
Dvc=Hostname_s,
DvcHostname=Dvc,

Reply:
Hi Varun,

Yes, that change was intentional. During the tests, there was a finding:
https://github.com/Azure/Azure-Sentinel/blob/1b01c1947417a14dadc3766771661699ca995f60/Parsers/ASimDns/Tests/Microsoft_DNS%20Server_Dns_SchemaTest_vimDnsMicrosoftNXLog.csv#L108 
```
"(2) Info: Missing recommended alias [Hostname] aliasing non-existent column [SrcHostname]"
```
I decided that was fairly easy to take care of.
Since the section you highlighted is part of a project-rename statement, it kind of naturally lends itself to have the `SrcHostname` created there.
For this reason, the creation of `Dvc` was moved to line 241, and since you can't reference a field in an extend statement that was created in the same statement, both `Dvc` and `DvcHostname` have to reference `SrcHostname`.
Furthermore, at the end, at line 278 I am also creating `Hostname` to be an alias of `SrcHostname`. 
That is the whole story.

Reply:
@jszigetvari-nxlog 
For DNS events, device fields refer to the system that reports the DNS event.
I think in this case, the following two old mappings are valid:
Dvc=Hostname_s
DvcHostname=Dvc

Please remove:
SrcHostname=Hostname_s
Dvc=SrcHostname,
DvcHostname=SrcHostname

sorry I missed this earlier. Looking into your other PR.

Reply:
What should be made with the `Hostname` alias? Should I tie it to `DvcHostname` or remove it altogether?

Reply:
its an alias to SrcHostname, since we don't have it. Lets remove Hostname too.

Reply:
@vakohl Okay, I undid those changes.